lib/iex/lib/iex/server.ex

```diff
@@ -313,4 +321,35 @@ defmodule IEx.Server do
   defp remote_prefix do
     if node == node(Process.group_leader), do: "iex", else: "rem"
   end
+
+  defp pry_code_lines(env) do
+    file = File.stream!(Path.relative_to_cwd(env.file))
+    code_start = pry_code_start_search(file, env.line)
+    code_end = pry_code_end_search(file, env.line)
+    code_list =
+      Enum.slice(file, code_start, code_end - code_start) |>
+      to_string
+    code_list = Regex.replace(~r/defp?/,code_list, fn x -> IO.ANSI.format([:green, :bright, x], true) end)
+    code_list = Regex.replace(~r/require IEx|IEx.pry/,code_list, fn x -> IO.ANSI.format([:blue, :bright, x], true) end)
+    Regex.replace(~r/end/,code_list, fn x -> IO.ANSI.format([:green, :bright, x], true) end) |>
+      IO.puts
+  end
```

Comment:
I've seen that IEx.Introspection checks `IEx.Config.ansi_docs` before printing docs with fancy colours. Maybe a similar check should be done here? I imagine a primitive terminal would spit out garbage on the colour escape codes otherwise.